bears/php/PHPCodeSnifferBear.py

```diff
@@ -0,0 +1,178 @@
+            Allows argument lists to be split accross multiple lines (correctly
+            indented.
+        '''
+        allow_tabs = ('<rule ref="Generic.WhiteSpace.DisallowTabIndent"/>'
+                      if use_spaces else '')
+        statements_per_line = (
+            '<rule ref="Generic.Formatting.DisallowMultipleStatements"/>'
+            if not allow_multiple_statements_per_line else '')
+        keywords_case = (
+            '<rule ref="Generic.PHP.LowerCaseKeyword"/>'
+            if force_lower_case_keywords else '')
+        constants_case = (
+            '<rule ref="Generic.PHP.LowerCaseConstant"/>'
+            if force_lower_case_constants else '')
+        namespace_declaration = (
+            'rule rel="PSR2.Namespaces.NamespaceDeclarationSniff"/>'
+            if blank_line_after_namespace_declaration else '')
+        use_declaration = (
+            'rule rel="PSR2.Namespaces.UseDeclarationSniff"/>'
+            if check_use_blocks else '')
+        class_declaration = (
+            'rule rel="PSR2.Classes.ClassDeclarationSniff"/>'
+            if check_class_declaration else '')
+        property_declaration = (
+            'rule rel="PSR2.Classes.PropertyDeclarationSniff"/>'
+            if check_property_declaration else '')
+        method_scope = (
+            '<rule ref="Squiz.Scope.MethodScope"/>'
+            if force_scope_modifier_on_method else '')
+        multiline_function_declaration = (
+            '<rule ref="Squiz.Functions.MultiLineFunctionDeclaration"/>'
+            if allow_multiline_function_declaration else '')
+        configs = '''<?xml version="1.0"?>
+<ruleset name="Custom Standard">
+ <description>A custom coding standard</description>
+ <rule ref="Generic.Files.LineLength">
+  <properties>
+   <property name="lineLimit" value="{max_line_length}"/>
+  </properties>
+ </rule>
+ <rule ref="Generic.Files.LineEndings">
+  <properties>
+   <property name="eolChar" value="{line_ending_character}"/>
+  </properties>
+ </rule>
+ <rule ref="Generic.WhiteSpace.ScopeIndent">
+  <properties>
+   <property name="ignoreIndentationTokens" type="array"
+   value="T_COMMENT,T_DOC_COMMENT_OPEN_TAG"/>
+  </properties>
+  <properties>
+   <property name="indent" value="{indent_size}"/>
+  </properties>
+  <properties>
+   <property name="exact" value="true"/>
+  </properties>
+ </rule>
+ {use_spaces}
+ {statements_per_line}
+ {keywords_case}
+ {constants_case}
+ {namespace_declaration}
+ {use_declaration}
+ {class_declaration}
+ {property_declaration}
+ {method_scope}
+ <rule ref="Squiz.WhiteSpace.ScopeKeywordSpacing"/>
+ <rule ref="Squiz.Functions.FunctionDeclarationArgumentSpacing">
+  <properties>
+   <property name="equalsSpacing"
+   value="{function_declaration_argument_spacing}"/>
+  </properties>
+ </rule>
+ <rule
+ ref="Squiz.Functions.FunctionDeclarationArgumentSpacing.SpacingAfterHint">
+  <severity>0</severity>
+ </rule>
+ {multiline_function_declaration}
+</ruleset>
```

Comment:
as the template spans quite some code, you maybe want to use an external file you load on module import / type creation^^
but it's up to you :+1: